modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -99,7 +99,7 @@ Latest release: 11/22/2024.
 xref:attachment$secure-properties-tool.jar[Secure Properties Tool Jar file - Supports Java 8 and 11]. +
 Latest release: 09/14/2022.
 
-Note that the syntax has changed from earlier versions. To avoid issues, use the syntax and the JAR file provided in this article.
+Note that the syntax has changed from earlier versions. To avoid issues, use the syntax and the JAR file provided in this article. Also, the tool does not support the hash character `#` in the input text.
```

Comment:
```suggestion
Note that the syntax has changed from earlier versions. To avoid issues, use the syntax and the JAR file provided in this article. Also, the tool doesn't support the hash character `#` in the input text.
```